SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
+	add-appxpackage "$packagePath\DynamicDependency.DataStore.Msix\DynamicDependency.DataStore.msix"
+	add-appxpackage "$packagePath\DynamicDependencyLifetimeManager.Msix\DynamicDependencyLifetimeManager.msix"
+	add-appxpackage "$packagePath\PushNotificationsLongRunningTask.Msix\PushNotificationsLongRunningTask.msix"
```

Comment:
As-is this is a utility for testing Push Notifications. It's not general purpose e.g. if I'm testing AppLifecycle I don't need (or want) PushNotificationsLongRunningTask.msix installed. And there are others I'm interested in.

As-is this should be somewhere Push-specific e.g. `\test\PushNotifications\PushNotificationsTestPackages.ps1`

Reply:
I feel there should be a little option prompt in the beginning of the script to select which packages to install rather than hardcode these packages:

1) Install Packages for Push
2) Install Packages for Deployment
3) Uninstall packages for Push
3) Uninstall packages for deployment etc.

Reply:
Looks like PushNotifications is the only component with manual demo testing. It seems appropriate to move this to test/PushNotifications. If another component wants to follow this pattern, then they can select only the packages they need.